README.md

```diff
@@ -50,9 +50,9 @@ need to be manually installed. Example flow of installing Metal3 provider:
    level of the logging verbose with a positive integer number, ie. -v5.
 
    ```shell
-   clusterctl init --core cluster-api:v1.8.1 \
-       --bootstrap kubeadm:v1.8.1 \
-       --control-plane kubeadm:v1.8.1 -v5
```

Comment:
I would rather keep this doc to point to 1.8.0 only and not update it in every patch from now on.

Reply:
Yup, put 1.8.0 back or untouch it here at least.